src/FooterList/index.tsx

```diff
@@ -0,0 +1,39 @@
+import { breakpoint, color, fontWeight, space } from '@core-ds/primitives';
+import * as React from 'react';
+import styled from 'styled-components';
+import { Link } from '../Link';
+
+export type FooterListItemProps = React.PropsWithChildren<{
+   className?: string;
+}>;
+
+export const FooterListItem = styled.li<FooterListItemProps>`
```

Comment:
We can definitely remove props here. They are there because I defined other properties before, but later changed my mind and simplified the component. Actually I was going to make a second PR, adding docs and polishing all this stuff